CMakeLists.txt

```diff
@@ -211,6 +211,12 @@ if(${VELOX_FORCE_COLORED_OUTPUT})
   endif()
 endif()
 
+if(CMAKE_SYSTEM_NAME STREQUAL "Linux"
```

Comment:
nit: small comment on why latomic for linux ?

Reply:
On MacOS this isn't needed explicitly so I wanted to make sure there is no issue for MacOS using clang already (AppleClang)